tests/neg/i3246.scala

```diff
@@ -0,0 +1,4 @@
+class Test {
```

Comment:
I think we should also compile it under scala2 mode, since it only crashed in this mode

Reply:
OK. Can you make the change and merge the PR when done? Thanks!

Reply:
Sure. Will do